registry.json

```diff
@@ -14,6 +14,18 @@
     "listed": true,
     "version": "2.1.0"
   },
+  "System.Buffers": {
+    "listed": true,
+    "version": "4.4.0"
+  },
+  "System.Memory": {
+    "listed": true,
+    "version": "4.5.0"
+  },
+  "System.Numerics.Vectors": {
+    "listed": true,
```

Comment:
Not sure it is a good idea to make `System.Numerics.Vectors` a listed package. Its primary usage - HW accelaration vectors is not supported in Unity.

Reply:
Fair point, much of what it offers is covered by Unity.Mathematics anyway. I'll change it to unlisted.